core/modules/dashboard/config/layout.layout.dashboard.json

```diff
@@ -7,7 +7,7 @@
     "renderer_name": "standard",
     "module": "dashboard",
     "weight": 0,
-    "storage": 4,
+    "storage": 2,
```

Comment:
Dont get why change this.

Reply:
Yeah, accidental I think. `4` means "default" storage, which it should be on new installs.